wgengine/magicsock/magicsock.go

```diff
@@ -430,6 +433,24 @@ type Conn struct {
 	peerLastDerp map[key.NodePublic]int
 }
 
+// SetDebugLoggingEnabled controls whether spammy debug logging is enabled.
+//
+// Note that this is currently independent from the log levels, even though
+// they're pretty correlated: debugging logs should be [v1] (or higher), but
```

Comment:
Why would we want these logs to be `[v1]` or above? Since they're not logged by default, turning on logging is (presumably?) a signal that we actually care about these and we want to see them... which we won't in any case if `tailscaled` isn't logging verbosely.